designate.md

```diff
@@ -0,0 +1,36 @@
+Deploying Designate
+-------------------
+
+Designate provides SUSE OpenStack Cloud, DNS as a service. It allows creating and propagating zones
```

Comment:
SUSE OpenStack Cloud uses Designate (DNS as a Service) to create and propgate zones and records over the network...